snark/src/lib.rs

```diff
@@ -38,34 +41,32 @@ pub trait SNARK<R: NPRelation> {
     /// Generates a proof of satisfaction of the arithmetic circuit C (specified
     /// as R1CS constraints).
     fn prove<Rng: RngCore + CryptoRng>(
-        circuit_pk: &Self::ProvingKey,
+        pk: &Self::ProvingKey,
```

Comment:
I feel like we should be more explicit with the naming here, e..g `proving_key`, since one could confuse it with a public_key / private_key

Reply:
doesn't the type clarify that?

Reply:
You're right, forgot it appeared in the docs